src/Interactive/HostTest/InteractiveHostTests.cs

```diff
@@ -118,14 +117,14 @@ private bool Execute(string code)
             return task.Result.Success;
         }
 
-        private bool IsShadowCopy(string path)
-        {
-            return _host.TryGetService().IsShadowCopy(path);
-        }
+        //private bool IsShadowCopy(string path)
+        //{
+        //    return _host.TryGetService().IsShadowCopy(path);
+        //}
 
-        public string ReadErrorOutputToEnd()
+        public async Task<String> ReadErrorOutputToEnd()
         {
-            return ReadOutputToEnd(isError: true);
+            return await ReadOutputToEnd(isError: true);
```

Comment:
>await [](start = 19, length = 5)

You can remove `async` and `await`. When you calling another method that is async and not doing anything with the result you can just return the Task it returns without awaiting it.